pkg/kubelet/prober/results/results_manager.go

```diff
@@ -87,3 +100,21 @@ func (m *manager) Remove(id kubecontainer.ContainerID) {
 	defer m.Unlock()
 	delete(m.cache, id)
 }
+
+func (m *manager) Updates() <-chan Update {
+	if m.updates == nil {
+		m.updates = make(chan Update)
+	}
+	return m.updates
+}
+
+// pushUpdates sends (non-blocking) an update on the updates channel if it is initialized.
+func (m *manager) pushUpdate(update Update) {
+	if m.updates != nil {
+		select {
+		case m.updates <- update:
+		default:
+			// Don't block
```

Comment:
The idea was that the updates were just triggering a sync, so they would only be dropped if a sync was already happening (in which case it was unnecessary). Now that I'll only be syncing individual pods, I think it makes sense for the worker to just block at this point.